### PR TITLE
Update Enable-PSRemoting help for PowerShell 6

### PR DESCRIPTION
**Pull request by PaulHigin**
# PR Summary
These changes update the Enable-PSRemoting help file to remove unneeded references to older versions of Windows PowerShell, and to add content that explains new behavior for PowerShell Core.

Issue #3109

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7 content
- [x] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md [hunk 1]
```diff
@@ -97,13 +121,68 @@ remote access from private and domain networks. The command uses the **SkipNetwo
...
+
+The first command enables PowerShell remoting on the computer.
+
+The second command lists the endpoint configurations.
+
+The third command creates a remote PowerShell session to the same machine, specifying the
+**PowerShell.6** endpoint by name. The remote session will be hosted with the latest PowerShell 6
+version (6.2.2).
+
+The last command accesses the **$PSVersionTable** variable in the remote session to display the
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md]: should be code fenced `PSVersionTable`

File: reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md [hunk 2]
```diff
@@ -21,19 +21,41 @@ Enable-PSRemoting [-Force] [-SkipNetworkProfileCheck] [-WhatIf] [-Confirm] [<Com
...
-
-Beginning in PowerShell 3.0, the `Enable-PSRemoting` cmdlet can enable PowerShell remoting
-on client versions of Windows when the computer is on a public network. For more information, see
-the description of the **SkipNetworkProfileCheck** parameter.
+have to run it on computers that only send commands. Because the configuration starts listeners to
+accept remote connections, it is prudent to run it only where it is needed.
+
+Enabling PowerShell remoting on client versions of Windows when the computer is on a public network
+is normally disallowed, but you can skip this restriction by using the **SkipNetworkProfileCheck**
+parameter. For more information, see the description of the **SkipNetworkProfileCheck** parameter.
```
Comment (chasewilson) [inline reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md]: You could link this to the parameter section if you wanted. e.g. [SkipNetworkProfileCheck](#-SkipNetworkProfileCheck)

double check that works before committing ;)

File: reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 external help file: System.Management.Automation.dll-Help.xml
 keywords: powershell,cmdlet
 locale: en-us
-ms.date: 10/02/2018
+ms.date: 1/10/2020
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md]: Use full-digit notation in date 01/10/2020

File: reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md (hunk 2 above)
Comment (PaulHigin) [reply reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md]: This change has caused a document validation warning.  So I will remove it.

**Pull request merged**